src/index.js

```diff
@@ -0,0 +1,102 @@
+const registerEvents = () => {
```

Comment:
This is acting more as like a main function entry point than a function that solely registers events. As a result, I might go with a name like `main` or `entryPoint`.